internal/version/version.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 )
 
+// Semantic version number parts.
```

Comment:
Perhaps we should check if the protobuf people have revised their corresponding code; that is, there is a TODO item and some links below in the `String()` function that maybe have been fixed? Didn't check.